### ISSUE TITLE
[JOSS Review] kraken.rules error

### ISSUE THREAD

**Issue by gbouras13** [kdm9/Acanthophis#13]
After fixing #12, I ran into this bug.

I tried to fix it, but it seems as though kraken.rules doesn't exist in the source code.

I had v0.2.0 of acanthophis installed via pip 

`pip install acanthophis `

```
  File "/Users/a1667917/mambaforge/envs/someproject/lib/python3.12/site-packages/snakemake/cli.py", line 1887, in args_to_api
    dag_api = workflow_api.dag(
              ^^^^^^^^^^^^^^^^^
  File "/Users/a1667917/mambaforge/envs/someproject/lib/python3.12/site-packages/snakemake/api.py", line 326, in dag
    return DAGApi(
           ^^^^^^^
  File "<string>", line 6, in __init__
  File "/Users/a1667917/mambaforge/envs/someproject/lib/python3.12/site-packages/snakemake/api.py", line 421, in __post_init__
    self.workflow_api._workflow.dag_settings = self.dag_settings
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/a1667917/mambaforge/envs/someproject/lib/python3.12/site-packages/snakemake/api.py", line 372, in _workflow
    workflow.include(
  File "/Users/a1667917/mambaforge/envs/someproject/lib/python3.12/site-packages/snakemake/workflow.py", line 1369, in include
    exec(compile(code, snakefile.get_path_or_uri(), "exec"), self.globals)
  File "/Users/a1667917/Documents/Reviews/acanthophis/Acanthophis/tests/Snakefile", line 24, in <module>
  File "/Users/a1667917/mambaforge/envs/someproject/lib/python3.12/site-packages/snakemake/workflow.py", line 1369, in include
    exec(compile(code, snakefile.get_path_or_uri(), "exec"), self.globals)
  File "/Users/a1667917/mambaforge/envs/someproject/lib/python3.12/site-packages/acanthophis/rules/kraken.rules", line 53, in <module>
KeyError: 'all'
```

**Comment by gbouras13** [comment]
For clarity, for issues #10-#13, I was using the pip install (v0.2.0).

George

**Comment by kdm9** [comment]
Ah, that would explain a few things. I hadn't tagged a new version in some time, because I was waiting for peer review before tagging a 1.0 release. This bug has been fixed, and I just tagged a 0.3.0 release that should contain the fix.

Apologies for my unclear instructions!
